linera-execution/src/policy.rs

```diff
@@ -61,6 +61,8 @@ pub struct ResourceControlPolicy {
     pub maximum_bytes_read_per_block: u64,
     /// The maximum data to write per block
     pub maximum_bytes_written_per_block: u64,
+    /// The maximum size of a received HTTP response.
+    pub maximum_http_response_size: u64,
```

Comment:
Can you add a unit to both the name and comment?

Reply:
Done.